drivers/bridge/setup_ip_tables.go

```diff
@@ -171,3 +173,51 @@ func setIcc(bridgeIface string, iccEnable, insert bool) error {
 
 	return nil
 }
+
+// Control Inter Network Communication
+func setINC(network1, network2 string, enable bool) error {
+	var (
+		table  = iptables.Filter
+		chain  = "FORWARD"
+		args   = [2][]string{{"-s", network1, "-d", network2, "-j", "DROP"}, {"-s", network2, "-d", network1, "-j", "DROP"}}
+		buffer bytes.Buffer
+	)
+
+	if enable {
+		for i := 0; i < 2; i++ {
+			if iptables.Exists(table, chain, args[i]...) {
+				continue
+			}
+			if output, err := iptables.Raw(append([]string{"-I", chain}, args[i]...)...); err != nil {
+				buffer.WriteString("unable to add inter-network communication: ")
+				buffer.WriteString(err.Error())
+				buffer.WriteString("; ")
+			} else if len(output) != 0 {
+				buffer.WriteString("error adding inter-network communication rule: ")
+				buffer.WriteString(string(output))
+				buffer.WriteString("; ")
+			}
+		}
+	} else {
+		for i := 0; i < 2; i++ {
+			if !iptables.Exists(table, chain, args[i]...) {
+				continue
+			}
+			if output, err := iptables.Raw(append([]string{"-D", chain}, args[i]...)...); err != nil {
+				buffer.WriteString("unable to remove inter-network communication rule: ")
+				buffer.WriteString(err.Error())
+				buffer.WriteString("; ")
+			} else if len(output) != 0 {
+				buffer.WriteString("error removing inter-network communication rule: ")
+				buffer.WriteString(string(output))
+				buffer.WriteString("; ")
+			}
+		}
+	}
+
+	if buffer.Len() != 0 {
```

Comment:
It will be done in the caller. This code was already designed for that as it first check for the rule presence before honoring a add/delete